### ISSUE TITLE
How to save character?

### ISSUE THREAD

**Issue by blockbreaker177** [jigsaw-4277821/Psych-Engine-0.5-Android-Port#6]
### Describe your problem here.

How do I save a character because I don't want to do the animation offsets again

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

**Comment by ImaginationSuperHero528** [comment]
From what I know the save and load buttons don't work...

**Comment by CalypsoTH** [comment]
> From what I know the save and load buttons don't work...

Yeah the only way to make a custom character is to pretty much find some base game character json, copy it and modify it.

**Comment by ImaginationSuperHero528** [comment]
> > From what I know the save and load buttons don't work...
> 
> Yeah the only way to make a custom character is to pretty much find some base game character json, copy it and modify it.

Or use the pc version which I have windows
